src/Nancy/ModelBinding/BindingConfig.cs

```diff
@@ -43,5 +43,11 @@ public BindingConfig()
         /// </summary>
         /// <value><see langword="true" /> if the binder is allowed to overwrite non-default values, otherwise <see langword="false" />.</value>
         public bool Overwrite { get; set; }
+
+        /// <summary>
+        /// If the string passed in is empty the binder will not bind it.
+        /// </summary>
+        /// <value><see langword="true" /> if the binder will convert empty strings to null, otherwise <see langword="false" />.</value>
+        public bool ConvertEmptyStringToNull { get; set; }

```

Comment:
I can just tell you that when I went searching for the problem the only thing I could think of searching for was ConvertEmptyStringToNull because of that being in the ASP.NET MVC project. It seems to have the same behavior. I agree that your name is probably better suited, and after the change there probably won't be anyone searching for it anymore maybe.

I just wonder how no one ran into this before?